tests/bulk_create/tests.py

```diff
@@ -199,6 +200,19 @@ def test_bulk_insert_expressions(self):
         bbb = Restaurant.objects.filter(name="betty's beetroot bar")
         self.assertEqual(bbb.count(), 1)
 
+    @skipUnlessDBFeature('has_bulk_insert')
+    def test_bulk_insert_nullable_fields(self):
+        # NULL can be mixed with other values in nullable fields
+        nullable_fields = filter(lambda field: field.name != 'id', NullableFields._meta.get_fields())
+        for i, field in enumerate(nullable_fields):
+            NullableFields.objects.bulk_create((
+                NullableFields(**{field.name: None, }),
+                NullableFields(),
+            ))
+            self.assertEqual(NullableFields.objects.count(), (i + 1) * 2)
```

Comment:
I'm not sure this assertion is needed. If anything, why not create the data first (perhaps in a single `bulk_create()`?), then have a separate loop for the assertions. I guess creating the data in batches helps identify if a field is broken.